src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -1,47 +1,120 @@
 // Copyright 2020 Google LLC
-
+//
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-
+//
 //     https://www.apache.org/licenses/LICENSE-2.0
-
+//
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package com.google.sps.servlets;
 
 import java.io.IOException;
-import javax.servlet.annotation.WebServlet;
-import java.util.Arrays;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.Collectors;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import com.google.gson.Gson;
 import com.google.sps.data.Meal;
+import java.util.HashMap;
 
-@WebServlet("/meal")
+@WebServlet("/meal/*")
 public class MealServlet extends HttpServlet {
 
-  Meal meal;
+    private final HashMap <Long, Meal> dishes = new HashMap <Long, Meal> ();
```

Comment:
Please remove the spaces around the `<>` brackets if not needed, i.e. instead of:  
`private final HashMap <Long, Meal> dishes`, please do:
`private final HashMap<Long, Meal> dishes`.